app/services/s3.js

```diff
@@ -31,7 +31,7 @@ class S3 {
         const createResponse = await this.s3.createMultipartUpload({
             Bucket: this.env.bucket,
             Key: this.key,
-            ContentType: "video/mp4",
+            ContentType: "video/webm",
         }).promise();
         this.uploadId = createResponse.UploadId;
```

Comment:
@okaycj yes that's right about containers only supporting specific codecs, and webm only supporting those codecs.
I'm now specifying webm along with those specific codecs (app/services/video-recorder.js, lines 197-201) but I see now that I have the wrong codec in by mistake (avc1 -> av1). I'll fix it, thanks!
And the reason I've change the content type here to `webm` is because we're not supporting other containers (so that we don't have to deal with changing this value and file extensions based on container type). Webm is available in Chrome and Firefox, which I think are the only browsers we officially support.

Reply:
Yup, I got a bit confused.  Nice that this comment wasn't completely useless.